src/collections/_documentation/workflow/integrations/integration-platform/ui-components.md

```diff
@@ -216,6 +238,68 @@ For a more complete description of the grammar, see the [source code](https://gi
     level="info"
 %}
 
+
+# Create/Link Issue
+
+If you have a UI component for issue linking and a user attempts to create or link an issue, it will send a request to your service based on the `uri` you provide.
+
+## Request Format
+
+The request that is sent is a `POST` request so all the information is stored in the body.
+
+**Schema**
+
+```
+{
+    "fields": <Object>,
+    "installationId": <String>,
+    "issueId": <String>,
+    "webUrl": <String>,
+    "project": {"slug": <String>, "id": <String>},
+    "actor": {"type": "user", "name": <String>, "id": <String>},
+}
+```
+
+**Attributes**
+
+- `fields` - Key/value pairs of the data in the form fields.
+- `installationId` - The ID of the installation associated with the request.
+- `issueId` - The ID of the issue in Sentry
+- `webUrl` - The URL of the issue in Sentry
+- `project`:
+   - `slug` - The slug of the project in Sentry
+   - `id` - The ID of the project in Sentry 
+- `actor`:
+   - `name` - The name of the user in Sentry
+   - `id` - The ID of user in Sentry 
+
+
+
+## Response Format
+
+When creating or linking an issue, the response format *must* have the following schema:
+
+
+**Schema**
+
+```
+{
+    "webUrl": <String>,
+    "project": <String>,
+    "identifier": <String>,
+}
+```
+
+
+**Attributes**
+
+- `webUrl`: The URL of the linked issue in your project management system.
+- `project`: The first part of the displayed issue that should probably be associated with a project in your system.
+- `identifier`: The second part displayed issue name that should be unique the project.
```

Comment:
```suggestion
- `identifier`: The second part displayed issue name that should be unique the project
```

Reply:
@scefali this description doesn't quite make sense. Do you mean "The second displayed issue name..."

Reply:
@MimiDumpling Whoops...I meant to say:
`The second part of the displayed issue that should be unique to the project.`